src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -66,6 +66,23 @@
  * <pre>
  * &lt;module name="NeedBraces"/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * if (true) { // OK
+ *     return true;
+ * } else // violation, single-line statements not allowed without braces
+ *     return false;
+ * for (int i = 0; i &lt; 5; i++) { // OK
+ *     ++count;
+ * }
+ * do // violation, single-line statements not allowed without braces
+ *     ++count;
+ * while (false);
+ * for (int j = 0; j &lt; 10; j++) // violation, single-line statements not allowed without braces
+ *                                 // violation, empty loop body not allowed
```

Comment:
> violation, empty loop body not allowed

please show by CLI that this violation is from Needbrace Check.

Reply:
It didn't show any error for empty loop body, opened an issue for same.